samples/subsys/mgmt/mcumgr/smp_svr/src/main.c

```diff
@@ -34,6 +34,18 @@
 #include <mgmt/smp_bt.h>
 #endif
 
+#if USE_PARTITION_MANAGER
+
+#include <pm_config.h>
+#define FLASH_AREA_STORAGE_ID       PM_LITTLEFS_STORAGE_ID
+
+#else
+
+#include <generated_dts_board.h>
```

Comment:
This should be `#include <devicetree.h>` according to upstream Zephyr, already gone through this :)

Reply:
Nevermind, will fix after merge.